test/functional/bitcoin_cli.py

```diff
@@ -17,10 +17,19 @@ def run_test(self):
         """Main test logic"""
 
         self.log.info("Compare responses from getinfo RPC and `bitcoin-cli getinfo`")
-        cli_get_info = self.nodes[0].cli.getinfo()
+        cli_get_info = self.nodes[0].cli().getinfo()
         rpc_get_info = self.nodes[0].getinfo()
-
         assert_equal(cli_get_info, rpc_get_info)
 
+        self.log.info("Test -stdinrpcpass option")
+        user, password = get_auth_cookie(self.nodes[0].datadir)
+        block_count = self.nodes[0].cli(rpcuser=user, rpcport=rpc_port(0), stdinrpcpass=None).getblockcount(input=password)
```

Comment:
I would prefer to make a temporary, empty datadir here instead of make it use the default. The tests should not touch `~/.bitcoin`.